tasks/php_apache.yml

```diff
@@ -126,6 +133,13 @@
       notify:
         - restart apache2
 
+    - name: LAMP | Install PHP libapache 
+      apt: name=libapache2-mod-php state=latest
```

Comment:
This is a bit strange, because I have clean deployments with existing role, where somehow this module is installed.
If this will play on 14.04 LTS, this should be ok, probably